lib/shopify-cli/helpers.rb

```diff
@@ -1,14 +1,15 @@
 module ShopifyCli
   module Helpers
-    autoload :AccessToken, 'shopify-cli/helpers/access_token'
```

Comment:
Thank you for alphabetizing!

Reply:
In vim you just have to highlight the list of things and run :!sort and it does it for me. 🤷‍♂

Reply:
Soon I will use vim